src/main/java/frc/robot/DistanceRange.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot;
+
+import java.util.HashMap;
+
+import frc.robot.AutonomousPath.VisionPathName;
+
+public class DistanceRange 
+  implements DistanceRangeInterface{
+
+    HashMap<String, Double> distanceRanges;
+
+    public DistanceRange(){
+        
+        this.distanceRanges = new HashMap<String, Double>();
+
+        distanceRanges.put("PathRedAUpperBound", 10.00);
+        distanceRanges.put("PathRedALowerBound", 8.50);
+        
+        distanceRanges.put("PathBlueAUpperBound", 16.30);
+        distanceRanges.put("PathBlueALowerBound", 16.10);
+
+        distanceRanges.put("PathRedBUpperBound", 10.40);
+        distanceRanges.put("PathRedBLowerBound", 10.30);
+
+        distanceRanges.put("PathBlueBUpperBound", 17.80);
+        distanceRanges.put("PathBlueBLowerBound", 17.50);
+    }
+
+    public VisionPathName getPathForCalculatedDistance(double distanceCalculated){
+
+        AutonomousPath objectAutonomousPath = new AutonomousPath();
+
+        if (distanceCalculated >= getDistanceRanges().get("PathRedALowerBound") && 
+              distanceCalculated <= getDistanceRanges().get("PathRedAUpperBound")){
```

Comment:
this block is still messy, and the whole distancerange approach was designed to eliminate it